src/mock_server.rs

```diff
@@ -239,7 +239,11 @@ impl Drop for MockServer {
     fn drop(&mut self) {
         debug!("Verify mock expectations.");
         if !run!(self.verify()) {
-            panic!("Verification failed: mock expectations have not been satisfied.");
+            if std::thread::panicking() {
+                debug!("Verification failed, but already panicking.");
```

Comment:
I would use the very same log message, I don't think we need to change it.
Debug sounds good as logging level.

Reply:
:+1: